example/project-rails-tailwind/test/components/stories/card_component_stories.stories.json

```diff
@@ -0,0 +1,49 @@
+{
+  "title": "Organisms",
+  "stories": [
+    {
+      "name": "card",
+      "parameters": {
+        "server": {
+          "id": "card_component/card"
+        }
+      },
+      "args": {
```

Comment:
It doesn't look like the user can currently change the button text, or any of the other button attributes (variant, arrows, etc.)

Reply:
TY!